crossdock/main.go

```diff
@@ -111,6 +111,7 @@ func httpHealthCheck(logger *zap.Logger, service, healthURL string) {
 	for i := 0; i < 240; i++ {
 		res, err := http.Get(healthURL)
 		if err == nil && is2xxStatusCode(res.StatusCode) {
+			res.Body.Close()
```

Comment:
fine here because it's a test code, but strictly speaking we must close when err==nil regardless of status code

Reply:
Yeah, I wanted to minimize the diff here